src/CMakeLists.txt

```diff
@@ -9,7 +9,7 @@ set(VERSION_SCRIPT_LOCATION "${CMAKE_CURRENT_SOURCE_DIR}/version-script.lds")
 macro(install_mc_rtc_lib_common MC_LIB KWD)
   target_include_directories(${MC_LIB} INTERFACE $<INSTALL_INTERFACE:include> $<BUILD_INTERFACE:${MC_RTC_INCLUDE_DIR}>)
   if(NOT EMSCRIPTEN)
-    if(WIN32)
+    if(MSVC)
```

Comment:
Actually this one should remain `WIN32` because the definitions are required due to the inclusion of `Windows.h` on WIN32

Reply:
Fixed in amended version of the commit